samigo/samigo-app/src/webapp/jsf/delivery/assessmentDeliveryHeading.jsp

```diff
@@ -45,12 +45,9 @@ Headings for delivery pages, needs to have msg=DeliveryMessages.properties, etc.
     </h:commandLink>
 
 <!-- SHOW FEEDBACK LINK FOR PREVIEW ASSESSMENT -->
-    <h:commandLink title="#{commonMessages.feedback}" action="takeAssessment" onmouseup="saveTime();" 
-       rendered="#{delivery.actionString=='previewAssessment' && !(delivery.pageContents.isNoParts && delivery.navigation eq '1')}" >
-     <h:outputText value="#{deliveryMessages.show_feedback}" />
-     <f:param name="showfeedbacknow" value="true" />
-     <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.delivery.DeliveryActionListener" />
-    </h:commandLink>
+<!-- SAM-3110 This link never seemed to work so disabled -->
```

Comment:
Careful comments like this are later rendered. Could use ui.remove if its really needed.